modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,7 +33,29 @@ This {release} version of the software and programming language should not be us
 
 == Highlights of what's new in {release}
 
-The {release} release primarily includes user-visible new features, internal enhancements, and fixes and updates to the version of {proglang} and the {proglang} base library that are supported. 
+The {release} release primarily includes internal enhancements, and fixes and updates to {proglang}, the {proglang} base library, and Candid. 
+
+The most significant new features and fixes include the following:
+
+* A fix to `+dfx upgrade+` that prevents a `+main+` thread panicked when `+Option::unwrap()+` is called on a `+None+` value. 
+* A fix to the name of the language service command to restore its compatibility with IDEs. 
+* Candid has been updated to support more Rust built-in types and with support for reference types.
+* The {proglang} compiler supports specifying command-line arguments using `+--args <file>+` and `+--args0 <file>+`.
```

Comment:
Can we prefix those next few items with `[compiler]` or `[proglang]`? "A change to option blocks and option checks..." is misleading that it might apply to DFX.